mix.exs

```diff
@@ -31,6 +31,7 @@ defmodule SGP30.MixProject do
   defp deps do
     [
       {:circuits_i2c, "~> 0.3"},
+      {:crc, "~> 0.10"},
```

Comment:
How would you feel about keeping as much native as possible and using `:cerlc` here instead? Or, maybe you have already gone through that decision and this was the result?

Reply:
Yeah, I hadn't seen that library! I'll make that swap

Reply:
@jjcarstens done!